packages/smarthr-ui/src/components/Icon/stories/Icon.stories.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+
+import { Cluster, Stack } from '../../Layout'
+import * as Icons from '../Icon'
+import { colorSet } from '../generateIcon'
+
+import type { Meta, StoryObj } from '@storybook/react'
+
+const FaAddressBookIcon = Icons.FaAddressBookIcon
+
+export default {
+  title: 'Media（メディア）/Icon',
+  component: FaAddressBookIcon,
+  render: (args) => <FaAddressBookIcon {...args} />,
+  argTypes: {
+    color: {
+      options: Object.keys(colorSet),
+    },
+    alt: { control: 'text' },
+    text: { control: 'text' },
+  },
+  args: {
+    color: 'TEXT_BLACK',
+    alt: '',
+    text: '',
+    iconGap: 0.25,
+    right: false,
+  },
+  parameters: {
+    chromatic: { disableSnapshot: true },
+  },
+} as Meta<typeof FaAddressBookIcon>
+
+export const All: StoryObj<typeof FaAddressBookIcon> = {
+  render: (args) => (
+    <Stack gap={0.75} as="ul" className="shr-list-none">
+      {Object.entries(Icons).map(([name, Icon]) => (
+        <li key={name}>
+          <Icon {...args} text={name.replace(/Icon$/, '')} />
```

Comment:
```suggestion
          <Icon {...args} text={args.text ?? name.replace(/Icon$/, '')} />
```

Playgroundがあるので :maa: ではあるんですが、Allページではargsのtextを優先してあげてもいいかな、という気持ちもあります

Reply:
👏